### PR TITLE
Add specifications list view

### PR DESCRIPTION
**Pull request by richard0000**
# What was done
### Design the specs list following the mockups.
- Prepared for the use case when there's no mapping in
place for the users's organization.
- And also for the use case when there' one or many
specs in place for the user's organization.

# Screenshots
specs-list

# Mockup
https://marvelapp.com/prototype/adf7283/screen/69635805

### CODE REVIEW

File: app/javascript/components/home/LeftCol.jsx [hunk 1]
```diff
@@ -12,9 +12,9 @@ class LeftSideHome extends React.Component {
             adipiscing elit, sed do eiusmod tempor incididunt ut labore et
             dolore magna aliqua.
           </p>
-          <button className="btn wide-btn btn-dark" type="button">
+          <Link to={"/specifications"} className="btn wide-btn btn-dark">
```
Comment (excelsior) [inline app/javascript/components/home/LeftCol.jsx:15]: Strings don't have to be evaluated in JSX, so curly braces aren't necessary here.

File: app/javascript/components/specifications-list/SpecList.jsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                    <th scope="col">
+                      <select className="form-control">
+                        <option value="user">Only My Mappings</option>
+                        <option value="all">All Mappings</option>
+                      </select>
+                    </th>
+                  </tr>
+                </thead>
+                <tbody>
+                  { specs.length > 0 ?
```
Comment (excelsior) [inline app/javascript/components/specifications-list/SpecList.jsx:34]: 0 is a falsey value in JS, so `> 0` is redundant.

**Review by excelsior — APPROVED** [review]
Looks good to me 👍

I'd suggest starting using ESLint and Prettier in order to enforce a consistent style in the code.

File: app/javascript/components/specifications-list/SpecList.jsx (hunk 2 above)
Comment (richard0000) [reply app/javascript/components/specifications-list/SpecList.jsx:34]: @excelsior it's to make the code self-explanatory. In this case, in my opinion, is easier to understand, by reading the code, that we want to print the list only when the `length > 0` condition is `true`.

I'm happy to change it anyway if you don't agree, but I'll wait for your input before merging.

**Pull request merged**